### ISSUE TITLE
Third party clients under active purge?

### ISSUE THREAD

**Issue by zmwangx** [SNH48Live/KVM48#15]
I was alerted to this thread http://tieba.baidu.com/p/6144846105.

![](https://user-images.githubusercontent.com/49120719/58480162-51d79c00-818c-11e9-9ba7-53ebbe8174df.png)

Not sure what to make of this. User agent is about the easiest thing to spoof in the world, I don't have a damn clue what's insurmountable about a user agent block, and with sufficient disguise how they could at all detect one is not using the official client, unless they encrypt/sign certain payloads — but then it's not a user agent problem. (Also, when you do any kind of encryption/signing/obfuscation client-side, the key has to be somewhere, whether embedded or requested on the fly, and the algorithm can always be reverse-engineered, it's only a matter of difficulty.)

Bottom line, I don't see any sign of KVM48 breaking down. I might need to be more diligent with updating the user agent, though.

Comments from anyone with more info is welcome.
